packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/exporters/ingress-nginx/network-policy.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rke2IngressNginx.networkPolicyEnabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  annotations:
+    np.rke2.io/ingress: resolved
+  name: rke2-ingress-network-policy
+  namespace: {{ .Values.rke2IngressNginx.namespaceOverride }}
```

Comment:
Seems like this doesn't actually have a default value. Is there supposed to be some sort of helper or lookup with this as an override when set? Or is this required? If required, we should add it as an empty (null) value and put a note that it should be set or something.

Reply:
It's there in `values.yaml`.
https://github.com/rancher/charts/blob/4411cc0ccaca933bb9a495413dd887d0922e9033/charts/rancher-monitoring/104.1.0%2Bup57.0.3/values.yaml#L335-L347

Reply:
So I think looking at other similar situations to this one, what we actually want to do here is:
1. define a new item in `_helpers.tpl` for this,
2. Call that helper here to reference that instead,
3. Allow the `values.yaml` to have the `namespaceOverride` value default be empty.

Reply:
So in this file we would replicate:
https://github.com/dharmit/rancher-charts/blob/aa9a6b007735706c311588752309cda89139982f/charts/rancher-monitoring/104.1.1-rc1%2Bup57.0.3/charts/prometheus-node-exporter/templates/daemonset.yaml#L5C26-L5C60

But call it: `"rke2-ingress-nginx.namespace"`

Reply:
And in the helpers file, I think we would mimic something like:
https://github.com/dharmit/rancher-charts/blob/aa9a6b007735706c311588752309cda89139982f/charts/rancher-monitoring/104.1.1-rc1%2Bup57.0.3/templates/_helpers.tpl#L301-L310

Since we know the default value should generally be `kube-system`, similar to how the default there is `kube-state-metrics`.